src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage.tsx

```diff
@@ -113,6 +113,11 @@ const TableToolPage: NextPage<TableToolPageProps> = ({
                   <Link
                     to="/data-tables/fast-track/[fastTrackId]"
                     as={`/data-tables/fast-track/${highlight.id}`}
+                    analytics={{
```

Comment:
Wondering if we even need the `analytics` prop on `Link`. I'm actually tempted to say we should just outright remove it. It's also bothering me that it's completely hijacking the `onClick` even as well (if you add an `onClick`, it won't fire)

Reply:
I didn't like how it was inconsistent - I'll create a ticket to remove it from Link.